[IAM-5212] fix docs for the new Okta SAML app according to Okta requirements (#1977)
saucelabs/sauce-docs: docs/basics/sso/configuring-sso-in-okta.md

3. Click **Add Integration**.
   <img src={useBaseUrl('img/basics/sso/idp-config/okta/add-integration.png')} alt="Add Integration" width="700" />

4. Enter the required information under **General Settings**, and then click **Next**.

> 5. On the **Sign On Options** page, in the **Metadata details** you find Identity Provider Metadata URL. Open it in a new tab. Copy the content and save as an XML file. Use this metadata file to set up the SSO integration at Sauce Labs in [Team Management Panel](/basics/sso/setting-up-sso#integrating-with-sauce-labs-service-provider).

Reviewer: ```suggestion
5. On the **Sign On Options** page, in the **Metadata details** section you can find the Identity Provider Metadata URL. Open it in a new tab, copy the content, and save it as an XML file. You can use this metadata file to set up the SSO Integration at Sauce Labs in the [Team Management Panel](/basics/sso/setting-up-sso#integrating-with-sauce-labs-service-provider).
```

Author: fixed

> 6. On the **Sign On Options** page in Okta, in the **Credentials Details** section make sure that the value **Application username format** is a [valid email address](/basics/sso/setting-up-sso/#name-id). The default format for **Okta username** is email address, unless it was changed in Okta Admin Console.

Reviewer: ```suggestion
6. On the **Sign On Options** page in Okta, in the **Credentials Details** section, ensure that the value for **Application username format** is a [valid email address](/basics/sso/setting-up-sso/#name-id). The default format for **Okta username** is an email address, unless it has been changed in Okta Admin Console.
```

Author: fixed
